src/main/java/com/arpnetworking/metrics/Quantity.java

```diff
@@ -24,15 +26,15 @@
 
     /**
      * Access the magnitude of the quantity.
-     * 
+     *
      * @return The magnitude.
      */
     Number getValue();
 
     /**
      * Access the units that the magnitude is expressed in.
-     * 
+     *
      * @return The units of the magnitude.
      */
-    Unit getUnit();
+    @Nullable Unit getUnit();
```

Comment:
There is some debate as to the usefulness of Optional. The Unit here was nullable previously and changing it to Optional would in fact change the contract. The @Nullable annotation simply clarifies the existing contract. I am happy to discuss this further with 2G; I've added a note to the design document to take this discussion up with 2G.